targets/gtk-extra

```diff
@@ -3,12 +3,15 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 REQUIRES='x11'
-DESCRIPTION='GTK-based tools including gdebi, gksu, and a simple browser.'
+DESCRIPTION='GTK-based tools including gdebi and a simple browser.'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
 install_dummy network-manager network-manager-gnome
-install gdebi gksu
+install gdebi
+if release -lt buster -lt kali-rolling; then
```

Comment:
This whitelist results in nothing on Ubuntu.  So you probably want to add `-lt artful`

Reply:
Done! 

Hmm. I imagine it still worked when I tested Xenial because gdebi requires gksu before debian ported it to policy kit. Would simply removing it from all targets be a better approach?